README.markdown

```diff
@@ -449,19 +449,20 @@ Keep short function declarations on one line including the opening brace:
 
 ```swift
 func reticulateSplines(spline: [Double]) -> Bool {
-  // reticulate code goes here
+    // reticulate code goes here
 }
 ```
 
 For functions with long signatures, put each parameter on a new line and add an extra indent on subsequent lines:
 
 ```swift
 func reticulateSplines(
-  spline: [Double], 
-  adjustmentFactor: Double,
-  translateConstant: Int, comment: String
+    spline: [Double], 
+    adjustmentFactor: Double,
+    translateConstant: Int, 
+    comment: String
```

Comment:
Not sure I necessarily agree with this as a hard rule.

In my opinion, `.   comment: String) -> Bool {` would be fine, too.